src/views/product-downloads-view/components/featured-tutorials-section/index.tsx

```diff
@@ -1,23 +1,18 @@
 import { ReactElement } from 'react'
-import { IconPlay16 } from '@hashicorp/flight-icons/svg-react/play-16'
-import { useCurrentProduct } from 'contexts'
-import getTruncatedText from 'lib/get-truncated-text'
-import { FeaturedTutorial } from 'views/product-downloads-view/types'
-import CardLink from 'components/card-link'
+import { FeaturedTutorialCard as FeaturedLearnCard } from 'views/product-downloads-view/types'
```

Comment:
🚧 Noticed that this had an alias added to it in a subsequent commit. Can we just rename the new `FeaturedTutorialCard` to `FeaturedLearnCard` in `src/views/product-downloads-view/types.ts` since it's more accurate?

Reply:
Absolutely, this was a slip up, VSCode's "rename symbol" isn't as good as I thought it was 😬 
✅ Should be fixed now!